runtime/browser/browser-vr-demo/browser-vr-demo.js

```diff
@@ -121,23 +121,9 @@ suggest(stage++);
 
 let chooseSuggestion = index => {
   document.querySelector('[particle-container]').textContent = '';
-  arc = cloneArc(arc); //arc.clone(); //prepareExtensionArc();
```

Comment:
The best version of that code is in another branch, I'll try to have it all unified this week.